### DISCUSSION TITLE
Add time tracker for todo's and projects

### DISCUSSION THREAD

**Discussion by Javx2023** [alainm23/planify#1629] [Ideas]
It's so helpful keep a track of a time expend on a job, so please may you add a time tracker for todo's with the possibility to summarize it on the hold project.

Thanks 
Keep such wondeful job you do on track.

**Comment by antrmn** [comment]
GNOME Hamster is a neat time tracker which includes a GNOME Shell panel widget, I highly suggest to check it out in the meantime.

Getting Things GNOME included a plugin which integrates with GNOME Hamster. Maybe Planify could follow this path.

**Comment by oleole39** [comment]
Another one also on GNOME:
- screenshots: https://flathub.org/apps/com.lynnmichaelmartin.TimeTracker
- code: https://github.com/elvishcraftsman/time-tracker
